meson.build

```diff
@@ -4012,6 +4013,7 @@ foreach tuple : sanitizers
                         if fuzz_tests
                                 test('@0@_@1@_@2@'.format(b, c, sanitizer),
                                      env,
+                                     suite : 'sanitizers',
```

Comment:
These are fuzz targets. They are just built with ASan/UBSan when `-Dfuzz-test=true` is passed to `meson`. I think they should go to the "fuzzers" suite as well.

Reply:
On the other hand if the idea is to the fuzz targets with `valgrind` using `--wrapper` they shouldn't be included in the "fuzzers" suite. I think maybe it would make sense to rename it to "fuzzers-sanitizers" or something like that. I'm not sure.

Reply:
I renamed it to `fuzzers+sanitizers`. Hopefully, that makes more sense.

Reply:
Thanks! Ideally this particular part of the systemd suite should be removed and Ubuntu CI limping along with no sanitizers whatsoever should be fixed to make it compatible with ASan/UBsan and run separate ASan/UBsan builds. That can't be addressed with a bunch of half-baked one-off patches though so probably it's never going to happen.

Reply:
Looks like `-Dfuzz-tests=true` is no longer used on Ubuntu CI so I guess it should be safe to remove that suite eventually. It's redundant when separate ASan/UBSan builds are run and have to be edited every time it's run in new environments by analogy with https://github.com/systemd/systemd/pull/22067/commits/2ced9f86ee237bc163ff11a22f9a71f39d10e479 anyway.

Reply:
This PR has the downside that meson prints the suite name in the test output:
```
915/1052 systemd:fuzzers / fuzz-varlink_array
308/314 systemd:fuzzers+sanitizers / fuzz-varlink_timeout-d8a88bf4adea54537d21e3afb396e1a55c5b58bf_address,undefined 
```

`systemd:fuzzers+sanitizers` is very wide again… Can we maybe make it narrower again? The name doesn't really need to be super precise.

Reply:
> Can we maybe make it narrower again?

@keszybz as long as the names are different I think it should be fine to rename that suite again. It's just that the fuzz targets built with ASan/UBSan can't be run with `--wrapper valgrind` along with the "fuzzer" suite.

Reply:
I don't understand your comment.

I was just asking for a string that is shorter than "fuzzers+sanitizers". If we can do that, that'd be great. If not, that's OK too.